azure-pipelines.yml

```diff
@@ -28,11 +28,8 @@ steps:
 
 - script: |
     go version
-    go get -v -t -d ./...
-    if [ -f Gopkg.toml ]; then
-        curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-        dep ensure
-    fi
+    git clone git@github.com:u-root/webboot
```

Comment:
This looks like it's using the latest code from head rather than the code in the PR.

Reply:
that's a problem. How do I fix that? and why did it work, hmm. ah because this head and that head are same, only azure gits changed. OK see next repo.